### DISCUSSION TITLE
Projection Globe / Mapbox GL 2.0

### DISCUSSION THREAD

**Discussion by alexpchin** [rnmapbox/maps#2128] [Q&A]
After reading the setup instructions and also looking through the library, it's still unclear to me whether it's possible to display the projection globe as per https://docs.mapbox.com/mapbox-gl-js/example/globe/

Can anyone assist with this?

**Comment by SkySails** [comment]
I would love for this to be featured in this wrapper! Looks like it is not supported at the moment though, but we can always hope for it to be included in the future!

**Comment by SkySails** [comment]
Added in #2294 🎉
